.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,11 +4,15 @@ Please fill in the following checklist, removing items that do not apply.
 See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
 -->
 
-- [ ] The page (if new), does not already exist in the repository.
-- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
-- [ ] The page has 8 or fewer examples.
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
-- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The page description includes a link to documentation or a homepage (if applicable).
+# Command info
 
-**Version of the command being documented (if known):**
+- **Version of the command being documented:** `latest`
+- **Brief description of the command being documented:** unspecified
+<!-- Replace `latest` with a concreete version when you know it exactly. -->
+
+- [ ] New command pages or translations are added. <!-- Check if you create a documentation page to tldr-pages. -->
+- [ ] All command pages are in correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.
```

Comment:
```suggestion
- [ ] The page(s) are in the correct platform directory: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.
```

I'd move away from specifying "command" pages since we can have alias pages as well for example.

Also, I'd move away from specifying "all" since a contributor should only be responsible that the page they're trying to add is in the correct directory, not that all pages are.

Ideally, there shouldn't be too many new pages across directories anyway, they'd be split into multiple PRs.

Reply:
I consider alias pages as a sort of command pages just referring to another pages. 😄 We can implicitly mean `all` here. Or we can rewrite it as: `At least one page is in the correct platform directory: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.` But in this case user doesn't guarantees that he/she/they tried to correctly place all pages.

Reply:
The Pages, suits better here.

The pages to me seems more like we are talking about the content being added 
whilst the 
All the new pages ... 
to me seems like a set of rules for all content, more like we are talking about the entire project rather than a users PR (I don't know how to explain it well, but thats the best I can up with ATM)

Reply:
Maybe we can go with "The pages" or "The page(s)" like before then, except plural?

Reply:
I've used `page(s)`.

Reply:
I agree with @SethFalco on this.